draft-ietf-quic-datagram.md

```diff
@@ -106,7 +106,7 @@ QUIC, such as for a Virtual Private Network (VPN). Internet-layer tunneling
 protocols generally require a reliable and authenticated handshake, followed by
 unreliable secure transmission of IP packets. This can, for example, require a
 TLS connection for the control data, and DTLS for tunneling IP packets. A single
-QUIC connection could support both parts with the use of unreliable datagrams.
+QUIC connection could support both parts when unreliable datagrams are available.
```

Comment:
I personally prefer the original text here, sorry.

Reply:
The motivation for any change here is that the original text sounds like the (reliable) control data is being supported by the unreliable datagrams, which seems unlikely to be the expected use.
Other options include "by augmenting reliable streams with unreliable datagrams", "could extend from the former to the latter by using unreliable datagrams", etc.

Reply:
How about `with the use of unreliable datagrams in addition to reliable streams`?

Reply:
sold